packages/cli/src/commands/sync-art.ts

```diff
@@ -1,47 +0,0 @@
-import { Arguments, CommandBuilder } from "yargs";
```

Comment:
@Kooshaba this is not used in sky strife anymore, is it?

Reply:
chatted on discord, seems like sky strife is actually still using this command, so we shouldn't remove it for now